### PR TITLE
Include key_type, value_type, title, and description in JsonSchemafier output

### LINKED ISSUE #41: Return title/description in JsonSchemafier
It would be useful to return the fields' title/description in JsonSchemafier

### LINKED ISSUE #42: Return nested schemas in JsonSchemafier
Return nested item schemas in JsonSchemafier. Currently when building a schema, we'll return the item schema for a sequence type (e.g. ListOrTuple), but without the underlying schema information for the contained type.

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #41 and fixes #42

### CODE REVIEW

**Review by jzuech3 — APPROVED** [review]
That looks perfect, thanks!

**Comment by jamadden** [comment]
1.11.0 is on PyPI.

File: src/nti/schema/jsonschema.py [hunk 1]
```diff
@@ -219,38 +222,66 @@ def make_schema(self):
...
+        item_schema['type'] = ui_type
+        item_schema['base_type'] = ui_base_type
+
+        if sch_interfaces.IChoice.providedBy(field):
+            choices, base_type = self.get_data_from_choice_field(field, ui_base_type)
+            item_schema['choices'] = choices
+            item_schema['base_type'] = base_type
+
+        if sch_interfaces.ICollection.providedBy(field):
+            item_schema['value_type'] = self._make_field_schema(field.value_type)
```
Comment (jzuech3) [inline src/nti/schema/jsonschema.py:281]: Arggh, looking at this closer now. I think we'll want to build the full schema out for the `field.value_type`. I think right here we are externalizing the `Object` when perhaps we want to store `JsonSchemafier(field.value_type).make_schema()`.

Comment (jamadden) [reply src/nti/schema/jsonschema.py:281]: You can't do that. `JsonSchemafier` takes a schema, not a field. `value_type` and `key_type` are fields.

What you could do is check for fields that are `IObject` and recurse into *their* `.schema` attribute, which should be a schema.

Comment (jzuech3) [reply src/nti/schema/jsonschema.py:281]: Yea, I believe you're correct. Sorry, when reviewing the PR, I thought it did what we needed.

**Comment by jamadden** [comment]
> Yea, I believe you're correct. Sorry, when reviewing the PR, I thought it
did what we needed.

No problem. I’m almost through the updates.

**Pull request merged**